boards/nordic/nrf54l20pdk/doc/index.rst

```diff
@@ -0,0 +1,99 @@
+.. _nrf54l15pdk_nrf54l15:
+
+nRF54L15 PDK
+############
+
+Overview
+********
+
+.. note::
+
+   All software for the nRF54L20 SoC is experimental and hardware availability
+   is restricted to the participants in the limited sampling program.
+
+The nRF54L20 Preview Development Kit hardware provides
+support for the Nordic Semiconductor nRF54L20 Arm Cortex-M33 CPU and
+the following devices:
+
+* CLOCK
+* RRAM
+* :abbr:`GPIO (General Purpose Input Output)`
+* :abbr:`MPU (Memory Protection Unit)`
+* :abbr:`NVIC (Nested Vectored Interrupt Controller)`
+* :abbr:`GRTC (Global real-time counter)`
+* :abbr:`UARTE (Universal asynchronous receiver-transmitter)`
+* :abbr:`WDT (Watchdog Timer)`
+
+Hardware
+********
+
+nRF54L20 PDK has two crystal oscillators:
+
+* High-frequency 32 MHz crystal oscillator (HFXO)
+* Low-frequency 32.768 kHz crystal oscillator (LFXO)
+
+The crystal oscillators can be configured to use either
+internal or external capacitors.
+
+Supported Features
+==================
+
+The ``nrf54l20pdk/nrf54l20/cpuapp`` board configuration supports the following
+hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| RRAM      | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| GRTC      | on-chip    | counter              |
++-----------+------------+----------------------+
+| SPU       | on-chip    | system protection    |
```

Comment:
what is the key of including peripherals in this table? It seems strange to include `SPU` and exclude `MPC` - they have similar role in the system (configuration of memory bus access protection).